app/assets/javascripts/admin/addon/routes/admin-route-map.js

```diff
@@ -236,6 +236,12 @@ export default function () {
             });
           }
         );
+        this.route("content", function () {
+          this.route("categoriesAndTags", { path: "/" });
+          this.route("sharing");
```

Comment:
is there a reason "sharing" doesn't have a path specified like the others here?

Reply:
By default the path will be inferred from the route name, but we don't want `/postsAndTopics`, we want `/posts-and-topics`, and for each section, we want one tab to be the default, pre-selected one, which is why we do `path: "/". 🙂